programs/svm-spoke/src/state/state.rs

```diff
@@ -7,7 +7,7 @@ pub struct State {
     pub paused_fills: bool,
     pub owner: Pubkey,
     pub seed: u64, // Add a seed to the state to enable multiple deployments.
-    pub number_of_deposits: u64,
+    pub number_of_deposits: u32,
```

Comment:
also need u32 in initial_number_of_deposits if initialize both in lib and admin.